package.json

```diff
@@ -4,7 +4,9 @@
   "version": "1.0.2",
   "author": "Jeff Dickey @dickeyxxx",
   "bugs": "https://github.com/heroku/cli-engine-config/issues",
-  "dependencies": {},
+  "dependencies": {
+    "cli-engine-config": "^1.0.3"
```

Comment:
Not sure. I thought it was something yarn did automagically. I'll remove it if it's not necessary.